cdxev/__main__.py

```diff
@@ -186,6 +186,13 @@ def create_amend_parser(
         help="Path to the SBOM file.",
         type=Path,
     )
+    parser.add_argument(
+        "--license-path",
+        metavar="<license-path>",
+        help="Path to the SBOM file.",
```

Comment:
change to 
`help="Path to a folder with txt-files containing license texts to be copied in the SBOM"`